src/Robo/Hooks/ExecuteInDrupalVmHook.php

```diff
@@ -31,7 +31,7 @@ public function executeInDrupalVm(ConsoleCommandEvent $event) {
         $event->disableCommand();
         $command = $event->getCommand();
         $new_input = $this->createCommandInputFromCurrentParams($command, $event->getInput());
-        $new_input->setOption('define', 'drush.alias=self');
```

Comment:
This is going to require a bit of testing. This is done because often the drush alias is not resolved directly inside of the virtual machine.

Reply:
I was concerned about this too. However, after some (very limited) testing, it appeared that local aliases worked just as well as `self` inside the VM. We should try to figure out why we decided to use self in the first place, and what the edge cases might be.